src/Summary/SummaryManager.php

```diff
@@ -123,4 +142,38 @@ public function removeLanguage(Adherent $adherent, Language $language): bool
 
         return true;
     }
+
+    public function setUrlProfilePicture(Summary $summary): void
+    {
+        $urlProfilePicture = '#';
```

Comment:
this is already defined on the entity, please use a constant if it's a static value, ou edit the settet
`setUrlProfilePicture($urlProfilePicture = '#')`